src/main/java/uk/gov/justice/probation/courtcasematcher/restclient/model/courtcaseservice/CCSExtendedCase.java

```diff
@@ -21,38 +24,45 @@
     private CCSDataSource source;
 
     public static CCSExtendedCase of(CourtCase courtCase) {
+        final var firstDefendant = courtCase.getFirstDefendant();
         return CCSExtendedCase.builder()
-                .caseId(courtCase.getCaseId())
+                .caseId(Optional.ofNullable(courtCase.getCaseId())
```

Comment:
Is this the case which is received ? We should not be keeping the case ID if so . we need the unique UUID-based one